recipes-graphics/wayland/weston-init.bbappend

```diff
@@ -49,3 +49,15 @@ do_install:append() {
         uncomment "$assignment" ${D}${sysconfdir}/xdg/weston/weston.ini
     done
 }
+
+WATCHDOG_SEC = "40"
+WATCHDOG_SEC_mx8ulp = "240"
+replace_setting() {
+    if ! grep -q "$1" $3; then
+        bbwarn "Setting to replace '$1' not found in file $3"
+    fi
+    sed -i -e "s,$1,$2," $3
+}
+do_install:append() {
+    replace_setting "WatchdogSec=.*" "WatchdogSec=${WATCHDOG_SEC}" ${D}${systemd_system_unitdir}/weston.service
+}
```

Comment:
This isn't a BSP thing; please move it to meta-freescale-distro and also restrict it for fsl distros.

Reply:
Done: https://github.com/Freescale/meta-freescale-distro/pull/80